core/shared/src/main/scala/fs2/Segment.scala

```diff
@@ -590,14 +590,14 @@ abstract class Segment[+O,+R] { self =>
 
   /**
    * Computes the result of this segment. May only be called when `O` is `Unit`, to prevent accidentally ignoring
```

Comment:
the comment is out of date, it should read "when `O` is `Nothing`"

Reply:
Thanks, fixed